site/content/en/latest/user/customize-envoyproxy.md

```diff
@@ -317,6 +317,36 @@ After applying the config, the bootstrap config will be overridden by the new co
 Any errors in the configuration will be surfaced as status within the `GatewayClass` resource.
 You can also validate this configuration using [egctl translate][].
 
+## Customize EnvoyProxy Horizontal Pod Autoscaler
+
+You can enable [Horizontal Pod Autoscaler](https://github.com/envoyproxy/gateway/issues/703) for EnvoyProxy Deployment via EnvoyProxy Config below:
```

Comment:
Should mention that we need a metrics-server installed in the cluster

Reply:
that's correct, let me update it